src/handler/http/models/alerts/mod.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 OpenObserve Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pub mod alerts;
+pub mod destinations;
+pub mod requests;
+pub mod responses;
+pub mod templates;
```

Comment:
The `alerts` `destinations` and `templates` modules contain structs that were copy-pasted from `config::meta::alerts`. Only structs and methods that are necessary for defining HTTP API-specific JSON de/serialization logic was copied over.

Although these structs are copy-pasted now - these structs and those defined in `config::meta::alerts` may eventually diverge from one another - though mapping functions are maintained in the `From` trait implementations to be able to map between the different layers.

Reply:
The `requests` and `responses` modules just define the actual expected request and response body schemas of each Alerts endpoint.